src/main/webapp/WEB-INF/pages/manage/autopublish.ftl

```diff
@@ -248,6 +248,14 @@
                         <div id="updateFrequencyTimeWrapper" class="col col-sm-3 col-md-2 col-lg-2 col-xl-1">
                             <input type="time" id="updateFrequencyTime" name="updateFrequencyTime" class="form-control" value="${updateFrequencyTime!"12:00"}">
                         </div>
+
+                        <div class="col-12 text-center mt-2">
```

Comment:
![Screenshot 2024-07-25 at 13 38 58](https://github.com/user-attachments/assets/41bb6b38-be9a-471f-bf3c-1bc3bf1abe7d)

Let's do this instead, that callout looks out of place